xbmc/music/MusicDatabase.cpp

```diff
@@ -1043,7 +1043,7 @@ bool CMusicDatabase::GetAlbum(int idAlbum, CAlbum& album, bool getSongs /* = tru
           songs.insert(idSong);
         }
 
-        int idSongArtist = record->at(songArtistOffset + artistCredit_idArtist).get_asInt();
+        record->at(songArtistOffset + artistCredit_idArtist).get_asInt();
```

Comment:
If the value is really not used then there's no need for the whole call as it's only purpose is to return a value for usage :) @DaveTBlake for checking why the value is not used.